hack/tools/internal/tilt-prepare/main.go

```diff
@@ -674,9 +686,13 @@ func kustomizeTask(path, out string) taskFunction {
 // workloadTask generates a task for creating the component yaml for a workload and saving the output on a file.
 // NOTE: This task has several sub steps including running kustomize, envsubst, fixing components for debugging,
 // and adding the workload resource mimicking what clusterctl init does.
-func workloadTask(name, workloadType, binaryName, containerName string, ts *tiltSettings, path string, getAdditionalObject func(string, []unstructured.Unstructured) (*unstructured.Unstructured, error)) taskFunction {
+func workloadTask(name, workloadType, binaryName, containerName string, ts *tiltSettings, path, options string, getAdditionalObject func(string, []unstructured.Unstructured) (*unstructured.Unstructured, error)) taskFunction {
 	return func(ctx context.Context, prefix string, errCh chan error) {
-		kustomizeCmd := exec.CommandContext(ctx, kustomizePath, "build", path)
+		args := []string{"build", path}
+		if options != "" {
+			args = []string{"build", options, path}
```

Comment:
I think this won't work if someone wants to pass in more than one arg to kustomize (as they have to be separate strings in l.695).

Maybe we should take a string array instead of a string pointer for options (like we did for e.g. extra_args.core)